### ISSUE TITLE
Apnea of prematurity

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#7440]
**Preferred term label:**
Apnea of prematurity

**Synonyms**
AOP

**Definition (free text, please give PubMed ID)**
Recurrent episodes of apnea occurring in infants born less than 37 weeks of gestation (PMID: 21301866). 

**Parent term (use hpo.jax.org/app)**
Apnea HP:0002104

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Comment**
AOP is specified by a pause of breathing for more than 15–20 s, or accompanied by oxygen desaturation (SpO2 ≤ 80% for ≥4 s) and bradycardia (heart rate < 2/3 of baseline for ≥4 s), in infants born less than 37 weeks of gestation (PMID: 21301866).

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

@pnrobinson

**Comment by jagcsun** [comment]
The term, Apneic episodes in infancy HP:0005949 looks similar but presence of Apnea in infants of **Premature birth** necessitated the request for creation of a new term, 'Apnea of prematurity'

**Comment by pnrobinson** [comment]
added
